src/core/pointcloud/qgspointcloudlayerrenderer.h

```diff
@@ -59,8 +64,8 @@ class CORE_EXPORT QgsPointCloudLayerRenderer: public QgsMapLayerRenderer
 
   private:
 
-    //! Traverses tree and returns all nodes in specified depth
-    QList<IndexedPointCloudNode> traverseTree( const QgsPointCloudIndex *pc, const QgsRenderContext &context, IndexedPointCloudNode n, float maxErrorPixels, float nodeErrorPixels );
+    //! Traverses tree and returns all nodes that intersects with a \a geometry in specified depth (if the geometry is null all nodes are considered)
```

Comment:
As far as i can see, the new geometry option isn't actually used anywhere. Is this a remnant from an earlier version?

Reply:
It is used here https://github.com/qgis/QGIS/blob/19afdbe2359159da871f7d22b252c766095a87d9/src/core/pointcloud/qgspointclouddataprovider.cpp#L236